scripts/create_istio_system.sh

```diff
@@ -31,3 +31,15 @@ else
   echo "running istioctl upgrade"
   istioctl upgrade -i $ISTIO_NAMESPACE -y ${ISTIO_FILES[@]/#/-f }
 fi
+
+if kubectl get namespace cattle-dashboards; then
+  kubectl delete configmap -n cattle-dashboards -l istio_dashboard=1
+  for dashboard in /usr/local/app/dashboards/*.json; do
+    name="istio-dashboard-$(basename ${dashboard} ".json")"
+    # replace datasource input placeholder in dashboard with datasource name in rancher-monitoring
+    sed -i 's/${DS_PROMETHEUS}/Prometheus/g' ${dashboard}
+    kubectl create configmap ${name} -n cattle-dashboards --from-file=$(basename ${dashboard})=${dashboard}
```

Comment:
Is this create going to error on upgrade if it already exists?

Reply:
It does:
```
k create configmap z
configmap/z created
k create configmap z
Error from server (AlreadyExists): configmaps "z" already exists
```

Reply:
would it be okay to cat the error and return zero exit code if its an already exists error?

Reply:
The script deletes all existing istio dashboards before recreating them (line 36, https://github.com/rancher/istio-installer/pull/13/files#diff-21bc9d08f8039851978731ed75c3e9d3544eb952f236996ce31ebdb73da2693cR36). This way we can also remove a dashboard during an upgrade.

Reply:
> would it be okay to catch the error and return zero exit code if its an already exists error?

Sure, I can add this. That would catch a case were a configmap with the name already exists that does not have the istio_dashboard=1 label.

Reply:
Done. Wrapped this in an if, so that it skips the configmap if it already exists and also does not add the istio_dashboard=1 label.

Reply:
Ahh missed that other place. Great.